packages/react-vapor/src/components/tab/Tab.tsx

```diff
@@ -10,6 +10,7 @@ export interface ITabOwnProps {
     title: string;
     disabled?: boolean;
     tooltip?: string;
+    children?: React.ReactNode;

```

Comment:
You don't need to redefined children in our interfaces, `React.Component`already declares the children prop.

Proof: on master branch you have autocomplete for it
![image](https://user-images.githubusercontent.com/35579930/102391859-47fecf00-3fa4-11eb-8b4d-0e9cd6c8ba32.png)
**